modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -282,7 +282,9 @@ export default function ListEntries({history}) {
 			show: ({canReassign}) => canReassign,
 		},
 		...useEntriesActions({
-			update: ({completed}) => completed === false,
+			update: ({assignees, completed}) =>
+				completed === false &&
+				assignees?.[0].id === Number(themeDisplay.getUserId()),
```

Comment:
@naraandrade i think if `assignees` is an empty array this condition will fail.

To prevent it, you can use this: `assignees?.[0]?.id`

Reply:
@albinojunior Ok, thanks for noticing, I will fix this :)

Reply:
Just a question: Should you apply the same condition on [ViewEntry](https://github.com/liferay/liferay-portal/blob/06e707df4454e36051558b8c5deaf7600497f9ab/modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/entry/ViewEntry.es.js#L206) page?

Reply:
Yes, I will apply this verification on ViewEntry too, thank you :)